### DISCUSSION TITLE
data stored in leveldb from the filer / using GRPC commands.

### DISCUSSION THREAD

**Discussion by william1benn** [seaweedfs/seaweedfs#4997] [Q&A] [answered]
How is the metadata encoded when stored in the filer-db, I would like to decode the the key/value from the filer when reading from leveldb/redis externally? Would it be a safe move to utilize this data directly outside of weed? For showing a UI count of files etc. 

Is there any docs, that maybe I missed in the wiki that gives more information on storing/retrieving files using GRPC? 

also possible to disable all UI's ?

**Comment by chrislusf** [comment, marked as answer]
* see https://github.com/seaweedfs/seaweedfs/blob/master/weed/filer/leveldb2/leveldb2_store.go#L84 for details.
* check filer.proto for gRPC APIs.
* Yes. See command line options.

**Reply by william1benn** [reply]
Thank you.
